packages/jss/src/utils/memoize.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Cache the value from the first time a function is called.
+ */
+const memoize = <Value>(fn: () => Value): (() => Value) => {
```

Comment:
1. just exporting it here is not making it public, you would have to export it from index
2. I think this function should stay internal, it's 3 lines of code and making it public is going to enforce maintenance on this and backwards compatibility

Reply:
maybe you were thinking to add atomic dom renderer to the core lib? I think it should for now stay a standalone package.

Reply:
Woo, yes I wanted to add `AtomicDomRenderer` sibling of `DomRenderer`

Reply:
The problem with this is that it will increase a bundle size while being optional.

Reply:
I mean in case someone  can't treeshake or for example bundlephobia is gonna show higher bundle size. I am actually planing on removing some stuff or moving it to separate package when it's not always used.

Reply:
I can help you for removing some stuff to reduce the package's size

Reply:
But now, I don't have any idea for separate stuff from `jss` package but I try to find

Reply:
I recommend to focus on creating the AtomicDomRenderer without doing any changes to jss itself, copy what you need first, then we will see if something needs to be done on jss side.

Reply:
Ok

Reply:
I will close this PR and continue the discussion on https://github.com/cssinjs/jss/issues/1462